### PR TITLE
Sdk nightly download mapper

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30826

## Description
Added the testing of upload and download to the sdk nightly

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: .circleci/config.yml
Comment (guyfreund) [inline .circleci/config.yml:1168]: Why deleted?

File: .circleci/config.yml
Comment (guyfreund) [inline .circleci/config.yml:1159]: update the name of the step in line `1149`

File: Tests/sdknightly/create_entities_for_nightly_sdk.py
Comment (guyfreund) [inline Tests/sdknightly/create_entities_for_nightly_sdk.py:136]: update description

**Review by guyfreund — CHANGES_REQUESTED** [review]
- Why do we have changes in `package.json` and `package-lock.json`?
- Please attach a successful sdk nightly run

File: Tests/sdknightly/create_entities_for_nightly_sdk.py
Comment (abaumgarten) [reply Tests/sdknightly/create_entities_for_nightly_sdk.py]: Done

File: .circleci/config.yml
Comment (abaumgarten) [reply .circleci/config.yml]: by mistake, adding it.

File: .circleci/config.yml
Comment (abaumgarten) [reply .circleci/config.yml:1159]: done

File: package.json
Comment (jochman) [inline package.json:23]: why is that?

**Review by jochman — CHANGES_REQUESTED** [review]
why you updated the mdx package?

File: package.json
Comment (abaumgarten) [reply package.json:23]: @guyfreund

Comment (abaumgarten) [reply package.json]: Done

**Comment by abaumgarten** [comment]
A successful nightly run : 
https://app.circleci.com/pipelines/github/demisto/content/46683/workflows/c21751cf-7e79-42f7-aa35-46587688094d/jobs/200164

**Review by guyfreund — APPROVED** [review]
Wait for @jochman's approval too

**Review by jochman — APPROVED** [review]

**Pull request merged**
